lib/rubocop-performance.rb

```diff
@@ -1,4 +1,4 @@
 # frozen_string_literal: true
 
 require 'rubocop'
-require_relative 'rubocop_performance/version'
+require_relative 'rubocop-performance/version'
```

Comment:
I think that `'rubocop/performance/version'` like RuboCop RSpec will be clean as a directory structure 😃 

```ruby
require_relative 'rubocop/performance/version'
```

- https://github.com/rubocop-hq/rubocop-rspec/blob/v1.27.0/rubocop-rspec.gemspec#L2
- https://github.com/rubocop-hq/rubocop-rspec/blob/v1.27.0/lib/rubocop/rspec/version.rb

Reply:
👍 I will update this PR to use that. Thanks

Reply:
I should have noticed this in the previous PR. Nice catch, @koic!